packages/shared/lib/services/sync/run.service.ts

```diff
@@ -158,7 +159,7 @@ export default class SyncRun {
         if (!nangoConfig) {
             const message = `No ${this.isAction ? 'action' : 'sync'} configuration was found for ${this.syncName}.`;
             if (this.activityLogId) {
-                await this.reportFailureForResults(message);
+                await this.reportFailureForResults(message, 0);
```

Comment:
I would be nice to use more json object, because when I read the code here it's impossible to know what 0 means

Reply:
Thanks, so much better 🙏🏻